build/Targets/Settings.props

```diff
@@ -30,7 +30,7 @@
     <VSIXExpInstallerPackageVersion>0.2.1-beta</VSIXExpInstallerPackageVersion>
     <RoslynDiagnosticsNugetPackageVersion>$(MicrosoftNetRoslynDiagnosticsVersion)</RoslynDiagnosticsNugetPackageVersion>
     <!-- Disable analyzers if NuGet deviated from the expected restore location (e.g. by changing case on a case-sensitive file system) -->
-    <UseRoslynAnalyzers Condition="!Exists('$(NuGetPackageRoot)\Microsoft.CodeQuality.Analyzers\$(RoslynDiagnosticsNugetPackageVersion)\analyzers\dotnet\cs\Analyzer.Utilities.dll')">False</UseRoslynAnalyzers>
```

Comment:
Can we just get rid of this hack now?

Reply:
This is tracked by https://github.com/dotnet/roslyn/issues/21005 and https://github.com/dotnet/roslyn/issues/21005#issuecomment-317822553 explains why it is not trivial work.